### PR TITLE
[1.1.1.1] Update the DNS over Discord docs

### PR DESCRIPTION
**Pull request by MattIPv4**
Updates the supported types for dig & multi-dig

Adds information (and GIFs) for the new refresh button on dig & multi-dig

Re-synchronizes the content with MattIPv4/DNS-over-Discord

### CODE REVIEW

**Comment by MattIPv4** [comment]
One note/question I had: I noticed in local dev with these changes that GIFs a left-aligned in the content, but PNGs are center-aligned. This isn't an actual issue, but looked a bit odd to me? Is there a recommended way to fix this, say by making all images left-aligned?

**Comment by MattIPv4** [comment]
👋 Hi @marciocloudflare, curious about what the timeline is likely to be for getting this reviewed/merged? :)

File: products/1.1.1.1/src/content/fun-stuff/dns-over-discord.md [hunk 1]
```diff
@@ -9,112 +9,228 @@ Ever wanted to make DNS requests from the comfort of your Discord server? Well n
...
-*   CERT
-*   CNAME
-*   MX
-*   NS
-*   SPF
-*   SRV
-*   TXT
+Discord has a limit of 25 options in slash commands, so DNS over Discord offers the 25 most common DNS record types to choose from:
+
+<table>
```
Comment (marciocloudflare) [inline products/1.1.1.1/src/content/fun-stuff/dns-over-discord.md]: Could you remove the table and leave this as a list?

File: products/1.1.1.1/src/content/fun-stuff/dns-over-discord.md [hunk 2]
```diff
@@ -9,112 +9,228 @@ Ever wanted to make DNS requests from the comfort of your Discord server? Well n
...
+
 When providing DNS record types for the `/multi-dig` command, Discord will not prompt you with options.
+
 Please provide a space-separated list of valid DNS record types to lookup, any invalid options will be silently dropped.
 'A' records will be used as the default if no valid types are given.
 
-![multi-dig command](../static/dns-over-discord/multi-dig-command.png)
+The following DNS record types are supported and considered valid by the bot:
+
+<table>
```
Comment (marciocloudflare) [inline products/1.1.1.1/src/content/fun-stuff/dns-over-discord.md]: Could you remove the table and leave it as a list?

**Review by marciocloudflare — COMMENTED** [review]
Hey @MattIPv4 . Could you remove the tables you added and leave the supported record types as they are for now? Thanks!

File: products/1.1.1.1/src/content/fun-stuff/dns-over-discord.md (hunk 1 above)
Comment (MattIPv4) [reply products/1.1.1.1/src/content/fun-stuff/dns-over-discord.md]: 👍 Sure can

File: products/1.1.1.1/src/content/fun-stuff/dns-over-discord.md (hunk 2 above)
Comment (MattIPv4) [reply products/1.1.1.1/src/content/fun-stuff/dns-over-discord.md]: 👍 Yup

**Review by marciocloudflare — APPROVED** [review]

**Comment by MattIPv4** [comment]
tyvm!

**Pull request merged**